FRENCH/src/ch07-03-paths-for-referring-to-an-item-in-the-module-tree.md

```diff
@@ -385,9 +386,9 @@ double-check why adding the `pub` keyword lets us use these paths in
 `add_to_waitlist` with respect to the privacy rules.
 -->
 
-Maintenant, le code va compiler ! Analysons les chemins relatifs et absolus et
+Maintenant, le code va compiler ! Analysons les chemins relatif et absolu et
```

Comment:
Since there is only one relative path and only one absolute path, their respective adjectives should be singular too, grammatically speaking (cf. [Adjectifs coordonnés](http://bdl.oqlf.gouv.qc.ca/bdl/gabarit_bdl.asp?id=1744)).

Reply:
OK, thank you for the explanation !